### DISCUSSION TITLE
Completely disable syntax highlighing?

### DISCUSSION THREAD

**Discussion by johnmapeson** [coteditor/CotEditor#1770] [Q&A] [answered]
Hello there. Could anybody tell, is it possible to completely disable syntax highlighting? For example, when I open a JavaScript file, I would prefer to see it without syntax highlighting.

I see there is "Settings > Format > Default syntax" thing, but it seems it doesn't affect syntax highlighting on file open.

<img width="547" alt="image" src="https://github.com/user-attachments/assets/ade6a846-3d73-48c7-89bd-504e05d0d5e3">

**Comment by 1024jp** [comment, marked as answer]
No. Unfortunately, CotEditor doesn't have any option to disable syntax highlighting for when opening existing files.

**Comment by karamfd** [comment]
@jsx97 You can use the mono theme and switch to another theme when you need syntax highlighting.
